pathologies.php

```diff
@@ -6,14 +6,14 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Pharmabase :: Pathologies</title>
+    <title>COJO :: Employés</title>
     <?php include "inc/resources.php" ?>
 </head>
 <body>
     <div class="wrapper">
         <header>
-            <h1>Pharmabase</h1>
-            <?php breadcrumb("Pathologies") ?>
+            <h1>COJO</h1>
+            <?php breadcrumb("Employés") ?>
```

Comment:
Je pense tu dois laisser ca a ```Pathodologies```, parceque, ```breadcums``` // function will take $_SERVER['REQUEST_URI'] and build a breadcrumb based on the user's current path.

Donc, j'assume ca refere au nom du fichier Pathodologies.